pages/_app.tsx

```diff
@@ -24,6 +42,8 @@ const Application = ({ Component, pageProps }: AppProps) => {
       </main>
 
       <Footer />
+
+      {analyticsScriptPath ? <Script src={analyticsScriptPath} /> : null}
```

Comment:
Never seen `next/script` before. Nicer than the react method I've used in the past: https://stackoverflow.com/questions/34424845/adding-script-tag-to-react-jsx

Reply:
Vercel is so thoughtful <3